go-controller/pkg/config/config.go

```diff
@@ -773,6 +788,12 @@ var CommonFlags = []cli.Flag{
 		Destination: &cliConfig.Logging.ACLLoggingRateLimit,
 		Value:       20,
 	},
+	&cli.IntFlag{
+		Name:        "egressip-reachability-timeout",
+		Usage:       "EgressIP node reachability timeout in seconds (default: 1)",
```

Comment:
This needs a better description I think. Is this a timeout per retry or in total for _all_ retries?

Reply:
total timeout